<ui version="4.0" >
 <class>PopupChatDialog</class>
 <widget class="QMainWindow" name="PopupChatDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="5" >
     <widget class="QTextBrowser" name="textBrowser" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item rowspan="3" row="0" column="5" >
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="avatarFrameButton" >
        <property name="minimumSize" >
         <size>
          <width>14</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>14</width>
          <height>31</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>509</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item rowspan="3" row="0" column="6" >
     <widget class="QFrame" name="avatarframe" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="avatarlabel" >
         <property name="minimumSize" >
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>121</width>
           <height>131</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>261</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" >
     <widget class="QPushButton" name="textboldButton" >
      <property name="minimumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset>C:/Dokumente und Einstellungen/Linux/.designer/backup</iconset>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QPushButton" name="textunderlineButton" >
      <property name="minimumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset>C:/Dokumente und Einstellungen/Linux/.designer/backup</iconset>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <widget class="QPushButton" name="textitalicButton" >
      <property name="minimumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset>C:/Dokumente und Einstellungen/Linux/.designer/backup</iconset>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4" >
     <widget class="QPushButton" name="colorButton" >
      <property name="minimumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5" >
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLineEdit" name="lineEdit" />
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="sendButton" >
        <property name="text" >
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
